core/src/main/java/io/forestframework/ext/core/AutoRoutingScanExtension.java

```diff
@@ -55,7 +55,8 @@ public void afterInjector(Injector injector) {
         RoutingManager routings = injector.getInstance(RoutingManager.class);
 
         // @formatter:off
-        List<Class<?>> componentClasses = injector.getInstance(Key.get(new TypeLiteral<List<Class<?>>>() { }, ComponentClasses.class));
+        List<Class<?>> componentClasses = injector.getInstance(Key.get(new TypeLiteral<List<Class<?>>>() {
```

Comment:
> [ant:checkstyle] [ERROR] /home/runner/work/forest-petclinic/forest-petclinic/core/src/main/java/io/forestframework/ext/core/AutoRoutingScanExtension.java:143:55: '{' is not preceded with whitespace. [WhitespaceAround]

The error says line 143, Why do you change this line?

Reply:
i swear i didnt touch this line, maybe i randomly touched the keyboard. SORRY, will double check.